test/api/justfile

```diff
@@ -2,7 +2,7 @@ default:
 
 
 clean-schema:
-    rm -rf {{justfile_directory()}}/smartphone-test-farm-client
+    rm -rf {{justfile_directory()}}/devicehub-client
 
 regen-schema: clean-schema
     poetry run openapi-python-client generate --overwrite --path $(git rev-parse --show-cdup)lib/units/api/swagger/api_v1.yaml --config openapi-gen-config.yaml
```

Comment:
Maybe specify [project_name_override and package_name_override](https://github.com/openapi-generators/openapi-python-client?tab=readme-ov-file#project_name_override-and-package_name_override) so that the name is consistant across everywhere: `devicehub`?